ui/shared/status/StatusChatInput.qml

```diff
@@ -187,20 +196,32 @@ Rectangle {
         insertInTextInput(messageInputField.selectionStart, wrapWith);
         insertInTextInput(messageInputField.selectionEnd, wrapWith);
         messageInputField.deselect()
+        formatInputMessage()
+    }
+
+    function formatInputMessage() {
+        const posBeforeEnd = messageInputField.length - messageInputField.cursorPosition;
+        const deparsedEmoji = Emoji.deparse(messageInputField.text);
+        const plainText = chatsModel.plainText(deparsedEmoji);
+        const formatted = parseMarkdown(Emoji.parse(plainText.replace(/\n/g, "<br />")))
+        messageInputField.text = formatted
+        messageInputField.cursorPosition = messageInputField.length - posBeforeEnd;
     }
 
     function onRelease(event) {
         // the text doesn't get registered to the textarea fast enough
         // we can only get it in the `released` event
         if (paste) {
             paste = false;
-            const posBeforeEnd = messageInputField.length - messageInputField.cursorPosition;
-            const deparsedEmoji = Emoji.deparse(messageInputField.text);
-            const plainText = chatsModel.plainText(deparsedEmoji);
-            messageInputField.text = Emoji.parse(plainText.replace(/\n/g, "<br />"));
-            messageInputField.cursorPosition = messageInputField.length - posBeforeEnd;
-
+            formatInputMessage()
             interrogateMessage();
+        } else {
+            if (event.key === Qt.Key_Asterisk ||
+                event.key === Qt.Key_QuoteLeft ||
+                event.key === Qt.Key_Space ||
```

Comment:
I think we also want to check, if it's indeed a `Key_Space`, whether there's a space before the cursor or not. If yes, we don't want to perform the format.

Due to how the format function is implemented, it's impossible right now to type multiple spaces.

I tried to determine whether there's a space before the cursor, but it didn't behave as expected. I wonder if @richard-ramos has some quick tips here, since there are some functions in this file that do deal with cursor positions as well